lib/tests/streamlit/caching/common_cache_test.py

```diff
@@ -162,6 +170,18 @@ def test_cached_st_function_warning(self, _, cache_decorator, call_stack):
         """Ensure we properly warn when st.foo functions are called
         inside a cached function.
         """
+        forward_msg_queue = ForwardMsgQueue()
+        orig_report_ctx = get_script_run_ctx()
+        add_script_run_ctx(
+            threading.current_thread(),
+            ScriptRunContext(
+                session_id="test session id",
+                enqueue=forward_msg_queue.enqueue,
+                query_string="",
+                session_state=SessionState(),
+                uploaded_file_mgr=None,
+            ),
+        )
```

Comment:
Nope. Removing the sharing test in the app_session test file caused the caching test to break because it was relying on this setup happening there. I don't fully understand how. But this change is to make it work again.